inst/rmarkdown/templates/asa_article/resources/template.tex

```diff
@@ -24,8 +24,13 @@
 %% load any required packages here
 $preamble$
 
+$if(highlighting-macros)$
+$highlighting-macros$
+$endif$
+
```

Comment:
Next time it will be better if you could separate this into a different pull request since it is not related to the new Springer format (i.e. [one thing per PR](https://yihui.name/en/2018/02/bite-sized-pull-requests/)).

Reply:
Right! (Didn't know it's not related.)